scripts/tests/test_bump_version.py

```
@@ -0,0 +1,30 @@
+import tomlkit
+from scripts.bump_version import _update_dependency_version
+
+
+def test_updates_string_dependency_preserving_prefix():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_updates_string_dependency_preserving_prefix`

Add return type annotation: `None`

(ANN201)